policykit/policyengine/views.py

```diff
@@ -658,3 +659,51 @@ def document_action_remove(request):
     action.save()
 
     return HttpResponse()
+
+def execute_policy(policy, action):
+    """
+    Execute policy for given action. This can be run repeatedly to check proposed actions.
+    """
```

Comment:
The idea is that `filter_policy`/`check_policy`/etc functions are never called outside of this function.